src/plugins/intel_cpu/tests/functional/CMakeLists.txt

```diff
@@ -64,10 +76,19 @@ else()
     set(TMP_EXPLICITLY_ENABLED_TESTS "${TMP_LIST_OF_EXPLICITLY_ENABLED_TESTS}")
 endif()
 
-if(NOT X86_64)
+if(RISCV64)
+    # temporary disable all custom tests for RISCV64
     list(APPEND EXCLUDED_SOURCE_PATHS
-         ${CMAKE_CURRENT_SOURCE_DIR}/custom/single_layer_tests/instances/x64
-         ${CMAKE_CURRENT_SOURCE_DIR}/custom/subgraph_tests/src/x64)
+        ${CMAKE_CURRENT_SOURCE_DIR}/custom/single_layer_tests
+        ${CMAKE_CURRENT_SOURCE_DIR}/custom/subgraph_tests)
+    # except ones which already enabled
+    file(GLOB_RECURSE TMP_LIST_OF_TEST_CLASSES          ${CMAKE_CURRENT_SOURCE_DIR}/custom/single_layer_tests/classes/*.cpp)
+    file(GLOB_RECURSE TMP_LIST_OF_COMMON_TEST_INSTANCES ${CMAKE_CURRENT_SOURCE_DIR}/custom/single_layer_tests/instances/common/*.cpp)
+    file(GLOB_RECURSE TMP_LIST_OF_COMMON_SUBGRAPH_TESTS ${CMAKE_CURRENT_SOURCE_DIR}/custom/subgraph_tests/src/common/*.cpp)
+    file(GLOB_RECURSE TMP_LIST_OF_SUBGRAPH_TEST_CLASSES ${CMAKE_CURRENT_SOURCE_DIR}/custom/subgraph_tests/src/classes/*.*)
+    list(APPEND TMP_LIST_OF_EXPLICITLY_ENABLED_TESTS
+        ${TMP_LIST_OF_TEST_CLASSES} ${TMP_LIST_OF_COMMON_TEST_INSTANCES} ${TMP_LIST_OF_COMMON_SUBGRAPH_TESTS} ${TMP_LIST_OF_SUBGRAPH_TEST_CLASSES})
+    set(TMP_EXPLICITLY_ENABLED_TESTS "${TMP_LIST_OF_EXPLICITLY_ENABLED_TESTS}")
```

Comment:
I want to define a function which only be used on x86_64 in file **src/plugins/intel_cpu/tests/functional/utils/filter_cpu_info.hpp**
```
#if defined(OPENVINO_ARCH_X86_64)
std::vector<CPUSpecificParams> filterCPUInfoForDeviceWithFP16(const std::vector<CPUSpecificParams>& allParams);
#endif
```
but I found it is not possible to do it because there is no RISCV64 configure in CMakeLists.txt. Currently on the master, RISCV64 is sharing most test code with x86_64. if I add function above, the RISCV64 CI will failed because there is no function  **filterCPUInfoForDeviceWithFP16** , so I think that I need to add configure in CMakeList.txt to limit which test cases will be tested on RISCV64

Reply:
alreadyalready revert the change in CMakeList.txt